src/powfacpy/case_studies.py

```diff
@@ -276,9 +280,10 @@ def filter_omitted_combinations(self, values_of_all_parameters_for_case, omitted
           if omitted_combination_dict[param_name] == "all":
             # If the parameter is 'all', then only one case should be created. The case where
             # the value of the parameter takes on the value of the first entry in 
-            # self.parameter_values[param_name][0] is selected. This must be the case for all
+            # original_parameter_values[param_name][0] is selected. This must be the case for all
```

Comment:
is original_parameter_values[param_name][0] a list or is this the first element? Then I would write  
'''
# takes on the value of the first entry in 
# original_parameter_values[param_name]
'''